packages/chat-client/src/controllers/engine.ts

```diff
@@ -356,10 +356,10 @@ export class ChatEngine extends IChatEngine {
     // NOTE: This is a very roundabout way to get back to symKey I by re-deriving,
```

Comment:
Looks like this note has become unattached from the original code. Can we remove please to avoid confusion 🙏